src/pages/index.js

```diff
@@ -26,13 +27,29 @@ const useStyles = makeStyles(theme => ({
     fontSize: 60,
     lineHeight: '71px',
     letterSpacing: '0.08em',
+    [theme.breakpoints.up('md')]: {
+      fontSize: 80,
+    },
   },
   diplomacyTextProperty: {
     color: '#fff',
     fontFamily: "'Rubik' , sans-serif",
     fontWeight: 'bold',
     fontSize: '24px',
-    lineHeight: '28px',
+    lineHeight: '58px',
+    [theme.breakpoints.up('md')]: {
+      fontSize: 35,
+    },
+  },
+  FontCalendar: {
+    [theme.breakpoints.up('md')]: {
+      fontSize: 30,
+    },
+  },
+  FontLocation: {
+    [theme.breakpoints.up('md')]: {
+      fontSize: 30,
+    },
   },
```

Comment:
Do you really need two classes with same properties for two different typography elements ??
Can't you apply the single class to multiple elements ??

Reply:
Sorry My mistake